__tests__/buildTranslationFiles.spec.ts

```diff
@@ -197,10 +218,11 @@ describe('buildTranslationFiles', () => {
       it('should work with service', () => {
         let expected = gKeys(19);
         expected['20.21.22.23'] = 'missing';
-        expected['24'] = 'missing';
```

Comment:
I see there are incorrect assignments, Can please replace all `'missing'` values with `m`? it should point to the variable instead of the value directly.

Reply:
renamed `m` to `defaultValue`.
replaced all occurrences of 'missing' with the new `defaultValue` variable, for a single source of truth.